@xen-orchestra/web/src/components/backups/logs/panel/cards/BackupLogResultsCard.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <UiCard class="card-container">
+    <UiCardTitle>
+      {{ cardMetadata.title }}
+      <UiCounter :value="results.length" size="small" :accent="cardMetadata.accent" variant="primary" />
+    </UiCardTitle>
+    <div class="content">
+      <template v-for="(result, index) in results" :key="index">
+        <VtsCardRowKeyValue>
+          <template #key>{{ t('message') }}</template>
+          <template #value>{{ result.message }}</template>
+          <template #addons>
+            <VtsCopyButton :value="result.message" />
+          </template>
+        </VtsCardRowKeyValue>
+        <UiLogEntryViewer
+          :label="t('api-error-details')"
```

Comment:
```suggestion
        <UiLogEntryViewer
           v-if="result.stack"
          :label="t('api-error-details')"
```

Sometimes there is no result and it's empty : 

<img width="371" height="93" alt="Image" src="https://github.com/user-attachments/assets/dc764cf6-32f2-4dcd-b61e-aa8fd2b7b667" />

But do we need this in info card ? 🤔

Reply:
And perhaps we can adapt the naming depending on the type?